client/src/components/ErrorMessageComponent/ErrorMessage.js

```diff
@@ -113,7 +125,10 @@ export class ErrorMessageComponent extends Component {
         isVisible={showModal}
         className="error-message">
         <div className="message-container">
-          <h2 tabIndex="0" className="heading-2" id="error-message-header"><span>!</span>Error</h2>
+          <h2 tabIndex="0" className="heading-2" id="error-message-header"><span>!</span>{this.props.t("errorMessages.error")}</h2>
+          { appAdtChinaUrl && <MessageBar>
```

Comment:
Remove the space between the bracket and variable name.

Reply:
Ready